packages/flutterfire_ui/lib/src/auth/widgets/error_text.dart

```diff
@@ -19,6 +19,9 @@ String? localizedErrorText(
       return labels.accessDisabledErrorText;
     case 'wrong-password':
       return labels.wrongOrNoPasswordErrorText;
+    case 'credential-already-in-use':
+      // TODO(lesnitsky): add translation
+      return 'This credential is already associated with a different user account.';
```

Comment:
This and other hardcoded strings are going to be addressed in a separate localisation PR